### PR TITLE
Es6 day 1

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone 1:Awesome books: with ES6

- Used ES6 modules to write modular JavaScript.
- Learned how to use proper ES6 syntax.
- Understood improvements that ES6 brings to JavaScript.
- Used npm as software packages system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+Microverse Community
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```
Comment (gemmen29) [inline README.md]: - [ ] Kindly fix the `mit` link in your README file.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
- [ ] Kindly use the es6 syntax for all functions in `date.js`, `hardworker.js`, and `navigation.js` to make your code more cleaner and professional, so it will be like:
```
const formatAMPM = (date) => {
}
```
- [ ] Kindly try to get rid of disabled linters lines in all files and fix them manually.

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
updated files, thanks for correction, please review my code,thankyou

**Review by Mre55 — APPROVED** [review]
**Hi @ibrahim777764,**

# Status: APPROVED 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**